pypeit/pypeit.py

```diff
@@ -566,84 +575,59 @@ def extract_one(self, frames, det, bg_frames=[], std_outfile=None):
         # For QA on crash
         msgs.sciexp = self.sciImg
 
-        # Object finding, first pass on frame without sky subtraction
+        # Instantiate Reduce object
         self.maskslits = self.caliBrate.tslits_dict['maskslits'].copy()
-
+        # Required for pypeline specific object
+        # TODO -- caliBrate should be replaced by the ~3 primary Objects needed
+        #   once we have the data models in place.
```

Comment:
You could try to replace the instantiate_me with the "get_instance" class method. To me that makes more sense since it is attached to the class and fewer lines of code.

Reply:
sounds like a good suggestion for your branch